File: js/bee1009.js
var input = require('fs').readFileSync('js/stdin', 'utf8');
var lines = input.split('\n');

var nome = lines[0]
var salarioBase = parseFloat(lines[1])
var bonus = parseFloat(lines[2])*0.15

var salarioFinal = (salarioBase + bonus)

console.log(`TOTAL = R$ ${salarioFinal.toFixed(2)}`)

File: C/bee1009.c
#include <stdio.h>
int main(){
    char nome[10];
    double salario,vendas,salarioFinal;
    scanf("%s", &nome);
    scanf("%lf", &salario);
    scanf("%lf", &vendas);
    salarioFinal = salario + (0.15*vendas);
    printf("TOTAL = R$ %.2f\n",salarioFinal);
}